### DISCUSSION TITLE
Is there a way to make preview not transparent?

### DISCUSSION THREAD

**Discussion by IvanKalinin** [atlassian/pragmatic-drag-and-drop#77] [Q&A]
The dragging preview, including the custom preview, is half transparent.
Is there a way to change opacity up to 100%?

**Comment by nmhungdev** [comment]
Duplicated: [#94](https://github.com/atlassian/pragmatic-drag-and-drop/issues/94#issuecomment-2244784611)

**Comment by hungify** [comment]
To address this issue, it is advisable to apply the technique outlined in the following guide: [renders a 1x1 element as the native drag preview](https://atlassian.design/components/pragmatic-drag-and-drop/core-package/adapters/element/drag-previews#non-native-custom-drag-previews).

Fortunately, I have completed this implementation. Check it out [here](https://github.com/atlassian/pragmatic-drag-and-drop/issues/162#issuecomment-2543457720).
